scope_indi.cpp

```diff
@@ -51,95 +51,103 @@
+        ISwitchVectorProperty *moveNS_prop;
+        ISwitch               *moveN_prop;
+        ISwitch               *moveS_prop;
+        ISwitchVectorProperty *moveEW_prop;
+        ISwitch               *moveE_prop;
+        ISwitch               *moveW_prop;
+        INumberVectorProperty *GuideRate_prop;
+        INumberVectorProperty *pulseGuideNS_prop;
+        INumber               *pulseN_prop;
+        INumber               *pulseS_prop;
+        INumberVectorProperty *pulseGuideEW_prop;
+        INumber               *pulseE_prop;
+        INumber               *pulseW_prop;
+        ISwitchVectorProperty *oncoordset_prop;
+        ISwitch               *setslew_prop;
+        ISwitch               *settrack_prop;
+        ISwitch               *setsync_prop;
+        INumberVectorProperty *GeographicCoord_prop;
+        INumberVectorProperty *SiderealTime_prop;
+        ITextVectorProperty   *scope_port;
+        ISwitchVectorProperty *pierside_prop;
+        ISwitch               *piersideEast_prop;
+        ISwitch               *piersideWest_prop;
+        ISwitchVectorProperty *AbortMotion_prop;
+        ISwitch               *Abort_prop;
+
+        wxMutex sync_lock;
+        wxCondition sync_cond;
+        bool guide_active;
+        GuideAxis guide_active_axis;
+
+        long     INDIport;
+        wxString INDIhost;
+        wxString INDIMountName;
+        bool     Connected {false};
+        bool     m_modal;
+        bool     m_ready;
+        bool     eod_coord;
+
+        bool     ConnectToDriver(RunInBg *ctx);
+        void     ClearStatus();
+        void     CheckState();
+
+    protected:
+        void newDevice(INDI::BaseDevice dp) override;
+        void removeDevice(INDI::BaseDevice dp) override;
+        void newProperty(INDI::Property property) override;
+        void removeProperty(INDI::Property property) override {}
+        void updateProperty(INDI::Property property) override;
+        void newMessage(INDI::BaseDevice dp, int messageID) override;
+        void serverDisconnected(int exit_code) override;
+
+    public:
+        ScopeINDI();
+        ~ScopeINDI();
+
+        bool     Connect() override;
+        bool     Disconnect() override;
+        bool     HasSetupDialog() const override;
+        void     SetupDialog() override;
+
+        MOVE_RESULT Guide(GUIDE_DIRECTION direction, int duration) override;
+        bool HasNonGuiMove() override;
+
+        bool   CanPulseGuide() override
+        {
+            return (pulseGuideNS_prop && pulseGuideEW_prop);
+        }
+        bool   CanReportPosition() override
+        {
+            return coord_prop ? true : false;
+        }
+        bool   CanSlew() override
+        {
+            return coord_prop ? true : false;
+        }
+        bool   CanSlewAsync() override;
+        bool   CanCheckSlewing() override
+        {
+            return coord_prop ? true : false;
+        }
+
+        double GetDeclination();

```

Comment:
This method renaming to GetDeclination() back from GetDeclinationRadians() breaks declination compensation as PHD2 doesn't get the correct declination anymore. Renaming this function back to GetDeclinationRadians() fixes it.

Reply:
Sure, made that in #1135